examples/demo.js

```diff
@@ -24,9 +24,6 @@ window.onload = function() {
     }
     // Null option
     addStepSelect.append('<option value="" disabled selected>Select a Module</option>');
-    addStepSelect.selectize({
```

Comment:
@Divy123 it is not clear to me why we no longer need this block, could you please explain?

Reply:
Actually this implemented search functionality but this is a big error causing agent and is not so good. So as of now I have fixed all the code and will implement a search functionality as soon as we find something apt.